CHANGELOGRELEASE.md

```diff
@@ -1,5 +1,26 @@
 ### V ...
 
+### V 0.29.0
+
+This release brings two additions to the interface. They should make it more easy to work with unpackaged genotype files.
+
+`trident genoconvert` gets the option `-o/--outPackagePath`, which allows to redirect the conversion output to any directory. If this option is used and if the input data is not read from unpackaged genotype data, then the POSEIDON.yml file of the source package is not updated.
+
+The second, more significant change is an additional interface option to input unpackaged genotype data. This affects the trident subcommands `init`, `genoconvert` and `forge`. Besides the verbose interface with `-r + -g + -s + -i`, it is now also possible to only give `-p/--genoOne` to fully describe one unpackaged genotype data set. `-p` takes one of the genotype data files (so `.bed`, `.bim` or `.fam` for PLINK or `.geno`, `.snp` or `.ind` for EIGENSTRAT) and determines based on its extension the data format (PLINK/EIGENSTRAT) and the paths to the other files forming the dataset (assuming they have the same name and are in the same directory).
+
+Coming back to the `forge` example below for `V 0.28.0`, we can now for example write:
+
+```
+trident forge \
+  -d 2017_GonzalesFortesCurrentBiology \
+  -r PLINK -g 2017_HaberAJHG/2017_HaberAJHG.bed -s 2017_HaberAJHG/2017_HaberAJHG.bim -i 2017_HaberAJHG/2017_HaberAJHG.fam \
```

Comment:
As an example this still looks a bit scary for a Changelog. I would suggest you also use the new `-p` feature in the first package case, so `-p 2017_HaberAJHG/2017_HaberAJHG.bed`.

Reply:
and for the record, following my more general API comment, this would then be `-p 2017_HaberAJHG/2017_HaberAJHG`

Reply:
Yes - that's a good point :+1: